src/Tools/BuildValidator/CompilationDiff.cs

```diff
@@ -5,12 +5,14 @@
 using System;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using System.Drawing.Text;
```

Comment:
I didn't see any new types referenced which obviously used these new usings.

Reply:
Had a few random `using` popup last night. Not sure why. Will remove.